core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -19,7 +19,7 @@ contract DesignatedVoting is Withdrawable {
      ****************************************/
 
     enum Roles {
-        Owner, // Can set the Voter and Withdrawer roles.
+        Owner, // Can set the Voter role. Is also permissioned to be the Withdrawer from initialization.
```

Comment:
I would make this more specific and say that the `Owner` and the `Withdrawer` are permanently the same role.